### PR TITLE
Snap builder

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
I added support for [Snap] packages. Our stable build is already available as `*.snap` [here](https://snapcraft.io/logisim-evolution). This PR updated nightly builder to also create `*.snap` packages. 

I disabled pushing such packages to `edge` channel on snapcraft.io as I am not sure this is needed.

* #743

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ include the Java runtime and do not require it to be installed separately:
 
 * `logisim-evolution_<version>-1_amd64.deb`: Debian package (also suitable for Ubuntu and derivatives),
 * `logisim-evolution-<version>-1.x86_64.rpm`: Package for Fedora/Redhat/CentOS/SuSE Linux distributions,
+* `logisim-evolution-<version>_amd64.snap`: The [Snap](https://snapcraft.io/docs) archive all
+  supported Linux distributions (also available in [Snapcraft store](https://snapcraft.io/logisim-evolution)),
```
Comment (maehne) [inline README.md:69]: "archive all" -> "archive for all"

**Review by maehne — APPROVED** [review]
Looks mostly good to me except for an incomplete sentence.

**Review by maehne — APPROVED** [review]
Looks good to me! Did you consider to upgrade the openjdk dependency to version 18?

**Comment by MarcinOrlowski** [comment]
LS-e needs JDK16, so I see no point upgrading.

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski : it seems that this PR broke the nightly build, which is not working any more in the moment. Do you have some time to look into this issue?

**Comment by MarcinOrlowski** [comment]
@BFH-ktt1 yep, will look into it

**Pull request merged**